src/Testing/TypeInferenceTestCase.php

```diff
@@ -125,7 +127,13 @@ public static function gatherAssertTypes(string $file): array
 			}
 
 			$functionName = $nameNode->toString();
-			if ($functionName === 'PHPStan\\Testing\\assertType') {
+			if (in_array($functionName, ['assertType', 'assertNativeType', 'assertVariableCertainty'], true)) {
```

Comment:
This should be case-insensitive + please also test it

Reply:
done